sample-client-app/sample-app-with-shared-lib/build-spring-boot.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+plugins {
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+	id 'java'
+}
+
+group = 'com.vmware.vipclient'
+version = '1.0'
+sourceCompatibility = '1.8'
+archivesBaseName = 'sample-spring-boot-with-shared-lib'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation files('lib/sample-shared-library-1.0.jar')
+	implementation files('lib/vip4java-0.1.0.jar')
```

Comment:
Why not download it from maven repo?

Reply:
We can take a look at this later in a separate PR as this has been like this before this PR.